### ISSUE TITLE
feature request: folder for playlist

### ISSUE THREAD

**Issue by XzuRukneg** [exaile/exaile#72]
Having sub-folder to organize my play list will be very helpful for me!

Thank you.

**Comment by virtuald** [comment]
What do you mean? Do you mean in the Playlist View that shows all the playlists?

**Comment by ghost** [comment]
Yes, folder(s) in the Playlist View that shows all the playlists.

**Comment by ghost** [comment]
A folder like the two already existing: "inteligent plylists" and "custom playlists" (from my translation!).
Will really help!

**Comment by ghost** [comment]
Nobody likes this request, right? :) I've got many custom playlists, for different use: for my student, for music studding, for me to listen, and so folder will help to find them quickly. Also because the order can change when I had a need one, so grouping is no possible.

**Comment by virtuald** [comment]
It's not a bad idea, just needs someone to do it.
